layouts/for-individuals/single.html

```
@@ -1,40 +1,159 @@
+  {{- $showAnniversaryBanner := .Scratch.Get "anniversaryBannerActive" -}}
+  {{- $showThinStrip := or $showWebinarBanner $showSaleBanner -}}
+  {{- $heroTopPad := "pt-16" -}}
+  {{- if and $showThinStrip $showAnniversaryBanner -}}
+    {{- $heroTopPad = "pt-40" -}}
+  {{- else if or $showThinStrip $showAnniversaryBanner -}}
+    {{- $heroTopPad = "pt-28" -}}
+  {{- end -}}
+  {{- /* The webinar banner hides itself client-side when no webinar is live/upcoming; collapse its reserved padding to match (mirrors baseof.html). */ -}}
+  {{- $webinarIsThinStrip := and $showWebinarBanner (not $showSaleBanner) -}}
+  {{- $heroPadNoStrip := cond $showAnniversaryBanner "pt-28" "pt-16" -}}
+  <header class="header-bg bg-dark bg-center bg-repeat-x relative{{ if not $webinarIsThinStrip }} {{ $heroTopPad }}{{ end }} md:pt-20 lg:pt-20"{{ if $webinarIsThinStrip }} :class="webinar.state === 'none' ? '{{ $heroPadNoStrip }}' : '{{ $heroTopPad }}'"{{ end }}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)